src/frontend/src/pages/ChangeRequestsPage/ChangeRequests.tsx

```diff
@@ -8,13 +8,16 @@ import { routes } from '../../utils/routes';
 import ChangeRequestDetails from '../ChangeRequestDetailPage/ChangeRequestDetails';
 import ChangeRequestsTable from './ChangeRequestsTable';
 import CreateChangeRequest from '../CreateChangeRequestPage/CreateChangeRequest';
+import ChangeRequestsOverview from './ChangeRequestsOverview';
 
 const ChangeRequests: React.FC = () => {
   return (
     <Switch>
+      <Route path={routes.CHANGE_REQUESTS_OVERVIEW} component={ChangeRequestsOverview} />\
+      <Route path={routes.ALL_CHANGE_REQUESTS} component={ChangeRequestsTable} />
       <Route path={routes.CHANGE_REQUESTS_NEW} component={CreateChangeRequest} />
       <Route path={routes.CHANGE_REQUESTS_BY_ID} component={ChangeRequestDetails} />
-      <Route path={routes.CHANGE_REQUESTS} component={ChangeRequestsTable} />
+      <Route path={routes.CHANGE_REQUESTS} component={ChangeRequestsOverview} />
```

Comment:
If we gonna set the route, /change-requests to return CR overview component, is there a reason we have to add an additional route, /change-requests/overview that returns the same CR overview component? Unless this is something that was asked by someone else, I want to confirm this with Sean and Reid if possible! :)

Reply:
I was following the example of how the Task List tabs work in the Project Detail pages—the page defaults to the "In Progress" view, but clicking on the actual tab redirects to "/in-progress" with the same view. If it makes more sense to get rid of this route no problem though